CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.27.0] - 2024-10-01
```

Comment:
Since this is just a bug fix and not a big change, I think we can just bump the patch number this time.

```suggestion
## [1.26.1] - 2024-10-01
```